### PR TITLE
rms-why-gplv3: 初稿翻译

### PR DESCRIPTION
**Pull request by biergaizi**

### CODE REVIEW

File: docs/rms-why-gplv3.md [hunk 1]
```diff
@@ -80,34 +121,60 @@ distributor of your copy was to add them. Tivoization is the way they
...
+GPLv3 仅仅在产品几乎完全被企业和机构独占时才对“机顶盒化”网开一面。
+
 Another threat that GPLv3 resists is that of patent deals like the
 Novell-Microsoft pact. Microsoft wants to use its thousands of patents
 to make users pay Microsoft for the privilege of running GNU/Linux, and
 made this pact to try to achieve that. The deal offers rather limited
 protection from Microsoft patents to Novell’s customers.
 
+GPLv3 所能抵御的另一个威胁就是像 Novell 和 Microsoft 这样的专利交易。
+微软希望通过它的数千项专利，被迫用户为获得运行 GNU/Linux 的特权而交费，
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]:
```
被迫用户为获得运行 GNU/Linux 的特权而交费，
^~~~
s/被迫/强迫
```

File: docs/rms-why-gplv3.md [hunk 2]
```diff
@@ -153,6 +247,11 @@ designated copyright holder, who will be in a position to upgrade the
 license version. One way or another, programs should provide this
 flexibility for future GPL versions.
 
+做到这一旦的方式之一就是将程序以“GPL 第三版或任何更新的版本”发布；另一个
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]:
```
做到这一旦的方式之一就是将程序以“GPL 第三版或任何更新的版本”发布...
     ^~~
error: potential type
note: did you mean `一切' ?
```

File: docs/rms-why-gplv3.md [hunk 3]
```diff
@@ -72,6 +105,14 @@ buying non-DRM players because none are available. No matter how many
 products you can choose from, they all have equivalent digital
 handcuffs.
 
+在那些被数字限制管理（DRM）—— 一个用来限制你使用自己电脑中数据的恶心功能 ——
+所控制的关键领域，市场竞争毫无作用，因为相关的竞争是被禁止的：《千年数字
+版权法案》（DMCA）和类似的法律下，在美国和许多国家，发布 DVD 播放器是非法的，
+除非播放器遵守由 DVD 利益团体制定的规则去限制用户（它们的网站是
+<http://www.dvdcca.org/>，但那些规则却似乎没有刊登在上）。公众们没有办法购买
+没有 DRM 的播放器将 DRM 拒之门外，因为市场上根本就没有啊。无论有多少产品可
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]:
```
公众们没有办法购买没有 DRM 的播放器将 DRM 拒之门外...
                             ^
warning: need refinement
note: 公众们没有任何办法购买没有 DRM 的播放器将 DRM 拒之门外
```

File: docs/rms-why-gplv3.md [hunk 4]
```diff
@@ -80,34 +121,60 @@ distributor of your copy was to add them. Tivoization is the way they
...
 Another threat that GPLv3 resists is that of patent deals like the
 Novell-Microsoft pact. Microsoft wants to use its thousands of patents
 to make users pay Microsoft for the privilege of running GNU/Linux, and
 made this pact to try to achieve that. The deal offers rather limited
 protection from Microsoft patents to Novell’s customers.
 
+GPLv3 所能抵御的另一个威胁就是像 Novell 和 Microsoft 这样的专利交易。
+微软希望通过它的数千项专利，被迫用户为获得运行 GNU/Linux 的特权而交费，
+而签订了这样一份协议来实现这个目标。这笔交易给 Novell 的消费者在微软的
+专利面前提供了相当有限的保护。
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]:
```
专利面前提供了相当有限的保护。
      ^
      仅
```

File: docs/rms-why-gplv3.md [hunk 5]
```diff
@@ -80,34 +121,60 @@ distributor of your copy was to add them. Tivoization is the way they
...
+
 Microsoft made a few mistakes in the Novell-Microsoft deal, and GPLv3 is
 designed to turn them against Microsoft, extending that limited patent
 protection to the whole community. In order to take advantage of this
 protection, programs need to use GPLv3.
 
+然而微软在与 Novell 的交易中犯了一些错误，而 GPLv3 旨在利用它们反击微软，
+排除这种对整个社区相当有限的专利保护。为了得益于这个保护措施，程序需要使
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]:
```
为了得益于这个保护措施
error: semantic error: `得益于' can't be used in this context, it is non-comprehensive.
```
- 为了从这个保护措施中获益
- 为了受益于这个保护措施

File: docs/rms-why-gplv3.md [hunk 6]
```diff
@@ -80,34 +121,60 @@ distributor of your copy was to add them. Tivoization is the way they
...
+用 GPLv3。
+
 Microsoft’s lawyers are not stupid, and next time they may manage to
 avoid those mistakes. GPLv3 therefore says they don’t get a “next time.”
 Releasing a program under GPL version 3 protects it from Microsoft’s
 future attempts to make redistributors collect Microsoft royalties from
 the program’s users.
 
+微软的律师们当然也不是傻子，它们下次的时候肯定会设法避免错误。因此，GPLv3
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]: 它们下次~~的时候~~肯定会设法避免错误

File: docs/rms-why-gplv3.md [hunk 7]
```diff
@@ -120,6 +187,14 @@ they implement ideas covered by hundreds of patents. Megacorporations
 collect thousands of patents, and use those patents to bully smaller
 developers. Patents already obstruct free software development.
 
+然而，GPLv3 中的专利保护并没有我们希望的那么强力。在理想的状态下，我们当然
+希望任何分发 GPL 许可软件的人，和不使用 GPL 许可证份发软件的人，都放弃他们
+全部的软件专利，因为软件专利根本就不应该存在。软件专利的体制是恶毒的、荒谬
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]: s/恶毒/罪恶

File: docs/rms-why-gplv3.md [hunk 8]
```diff
@@ -120,6 +187,14 @@ they implement ideas covered by hundreds of patents. Megacorporations
...
+的，将所有软件开发者都置于被一个闻所未闻的公司起诉的危险之下，更别说那些大
+型集团了。大型程序往往包含了成百上千的思想，因此这些思想被上百个专利所涵盖
```
Comment (liushuyu) [inline docs/rms-why-gplv3.md]: 因此这些思想被上百个专利所涵盖
- This sentence doesn't have direct reason-result logical relationship with/no obvious /explicit logical relationship with  the previous clause, therefore `因此` is not appropriate.
- `涵盖` is not appropriate within the given context.

**Comment by tonghuix** [comment]
亲，你最好直接提PR来修

**Pull request merged**